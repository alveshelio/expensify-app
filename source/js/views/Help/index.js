import React from 'react';

const Help = (props) => {
  console.log('props', props);
  return (
    <div>
      <h1>Help Page</h1>
    </div>
  );
};

export default Help;
