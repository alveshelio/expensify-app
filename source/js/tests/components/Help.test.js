/* eslint no-undef: "off" */

import React from 'react';
import { shallow } from 'enzyme';

import Help from '../../views/Help';

test('it should render Help page', () => {
  const wrapper = shallow(<Help />);
  expect(wrapper).toMatchSnapshot();
});
