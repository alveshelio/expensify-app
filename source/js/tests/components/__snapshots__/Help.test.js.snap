// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render Help page 1`] = `
<div>
  <h1>
    Help Page
  </h1>
</div>
`;
